modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,195 @@
+. Complete the initial setup according to xref:installation:proxy-setup.adoc[].
+. At the command prompt, run:
++
+----
+configure-proxy.sh
+----
+. At the ``Do you want to import existing certificates?`` prompt, type kbd:[y]
+. Follow the prompts to complete setup.
 
 
-[[bp.cert.custom.setup]]
-== Setup
 
+== Re-Create Existing Certificates
 
-After completing {yast}
-firstboot procedures, export your current environment variables and point them to the correct SSL files to be imported.
-Running these commands will make the default certificate obsolete after executing the [command]``yast2 susemanagersetup`` command.
-For more information on {yast}
- firstboot, see https://www.suse.com/documentation/suse-manager-3/singlehtml/suse_manager21/book_susemanager_install/book_susemanager_install.html#sec.manager.inst.setup.
-[[bp.cert.custom.setup.proc]]
+If your existing custom certificates have expired or stopped working for any reason, you can generate a new certificate from the existing CA.
 
+.Procedure: Re-Creating an Existing Certificate
 
-[[bp.cert.custom.setup.proc.export]]
-. Export the environment variables and point to the SSL files to be imported:
+. On the {productname} Server, at the command prompt, regenerate the server certificate:
 +
-
 ----
-export CA_CERT=`path_to_CA_certificate_file`export SERVER_KEY=`path_to_web_server_key`export SERVER_CERT=`path_to_web_server_certificate`
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+--set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+--set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+--set-hostname="susemanager.example.top" --set-cname="example.com"
 ----
-. Execute {productname} setup with
+Ensure that the [systemitem]``set-cname`` parameter is the fully-qualified domain name of your {productname} Server.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases.
+. Install the RPM that contains the newly generated certificate:
 +
-
 ----
-yast2 susemanagersetup
+rpm -Uhv /root/ssl-build/lnx0259a/rhn-org-httpd-ssl-key-pair-lnx0259a-1.0-2.noarch.rpm
 ----
+. Restart services to pick up the changes:
 +
-Proceed with the default setup.
-Upon reaching the Certificate Setup window during {yast}
-installation, fill in random values, as these will be overridden with the values specified in <<bp.cert.custom.setup.proc.export>>.
-+
-.Shell Requirements
-NOTE: Make sure that you execute [command]``yast2
-      susemanagersetup`` from within the same shell the environment variables were exported from.
-+
+----
+spacewalk-service restart
+----
 
 
 
-[[bp.cert.custom.proxy]]
-== Using a Custom Certificate with {susemgrproxy}
+== Create New Certificates
 
+If you need to create entirely new certificates for an existing installation, you will need to delete the old certificates before you begin.
```

Comment:
We should say that this is dangerous as it could happen that you by accident cut the trust chain from the clients to the server. If this happens, the admin need to login to every client and deploy the new CA there.